python/dgl/distributed/partition.py

```diff
@@ -1,10 +1,14 @@
 """Functions for partitions. """
 
+import concurrent
```

Comment:
is this really required? seems below one is sufficient.

Reply:
Seems like I missed this one, I can fix in a follow-up PR.